Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -25,9 +27,14 @@ class PetSingleDetailsViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setPlaceholderImage()
-        adoptPetButton.addCornerRadius()
-        saveSinglePetButton.changeBorderLook()
-        saveSinglePetButton.addCornerRadius()
+        self.title = singlePetViewModel.singlePetName
+        petImageView.layer.cornerRadius = 50
```

Comment:
Feel like this could also live in an extension 😄

Reply:
I agree 👍🏻